src/components/Alert/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ComponentType } from 'react';
+
+export type AlertProps = {
+  Icon: ComponentType;
+  dataTestId?: string;
+  alertHeader?: string;
+  errorMessage: string[];
+};
+
+const Alert = ({ Icon, dataTestId, alertHeader, errorMessage }: AlertProps): JSX.Element => {
```

Comment:
I'm wondering how is this different from the `AlertWarning` component 💭 
Also are we using this component anywhere?

Reply:
I've deleted this file we don't use it anymore 😀